### PR TITLE
Expose extra billing fields

### PR DESCRIPTION
**Pull request by stephencelis**
This allows folks to tack on `?expand=true` and enter their address and/or VAT number if required. It expands the "update payment info" form by default, though showing VAT (even with the "Optional" parenthetical) is admittedly confusing. I'm cool addressing later, but if you have a quick idea to fix, lemme know!

### CODE REVIEW

File: Sources/PointFree/Account/Account.swift [hunk 1]
```diff
@@ -293,7 +293,7 @@ private func mainAction(for subscription: Stripe.Subscription) -> Node {
     return a(
       [
         `class`([Class.pf.components.button(color: .purple, size: .small)]),
-        href(path(to: .pricing(nil)))
+        href(path(to: .pricing(nil, expand: nil)))
```
Comment (stephencelis) [inline Sources/PointFree/Account/Account.swift]: FWIW I was gonna just do `.pricing(Pricing, address: Bool?, vat: Bool?)` but we need to update our router to support more matches.

Comment (mbrandonw) [reply Sources/PointFree/Account/Account.swift]: whatdya mean more matches? it didn't work to do `queryParam("vat", opt(.bool))`?

Comment (stephencelis) [reply Sources/PointFree/Account/Account.swift]: Yeah I think our router can only match 3 times right now.

Comment (mbrandonw) [reply Sources/PointFree/Account/Account.swift]: it should be able to do more no prob, as long as there is a `parenthesize` defined for it:

https://github.com/pointfreeco/swift-web/blob/34b4143b4153f4b624364203e55e61e2fa9edc38/Sources/ApplicativeRouter/PartialIso.swift#L71

may have been something else

**Pull request merged**